lib/utils/config-helpers.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @fileoverview eslint config helpers
+ * @author 唯然<weiran.zsd@outlook.com>
+ */
+
+exports.extendsRules = function (config, rules) {
```

Comment:
I think this function should be called `extendRules`, not `extendsRules`. Also, please use `module.exports` instead of only `exports` to stay consistent with other files.

Reply:
will do! 👍  (I thought there would be more than one function to export,)

Reply:
sidenote: you may want to enable `exports-style` to enforce it.

https://github.com/eslint-community/eslint-plugin-n/blob/master/docs/rules/exports-style.md